chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/schedule/ChplTrigger.java

```diff
@@ -44,8 +39,10 @@
     public ChplTrigger(final CronTrigger quartzTrigger) {
         this.name = quartzTrigger.getKey().getName();
         this.group = quartzTrigger.getKey().getGroup();
-        this.jobName = quartzTrigger.getJobKey().getName();
-        this.jobGroup = quartzTrigger.getJobKey().getName();
+        this.job = new ChplJob();
+        this.job.setDescription("");
```

Comment:
Shouldn't the description be set to something?

Reply:
In this situation, we don't have access to the JobDetails object.  A CronTrigger object only has a reference to the JobKey.  We could use the ChplSchedulerReference singleton, to get the JobDetail based on the JobKEy.  Do you think this class should a reference to ChplSchedulerReference?

Reply:
I guess it doesn't really matter in any case as we don't use the Description of a trigger anywhere yet